modules/scripts/startup-script/files/setup-raid.yml

```diff
@@ -0,0 +1,74 @@
+# Copyright 2024 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+---
+
+- name: Configure local SSDs
+  become: true
+  hosts: localhost
+  vars:
+    raid_name: localssd
+    mountpoint: /mnt/{{ raid_name }}
+    array_dev: /dev/md/{{ raid_name }}
+    fstype: ext4
+    interface: nvme
+    mode: '0755'
+  tasks:
+  - name: Install mdadm
```

Comment:
Move this task after the end_play task so we are not unnecessarily installing `mdadm`.

Reply:
Done